api/v1alpha4/common_types.go

```diff
@@ -20,6 +20,11 @@ import (
 	"k8s.io/apimachinery/pkg/selection"
 )
 
+const (
+	// UnhealthyAnnotation is the annotation that sets unhealthy status of BMH
+	UnhealthyAnnotation = "capm3.metal3.io/unhealthy"
```

Comment:
capm3 and metal3 are a bit redundant here. what about "capi.metal3.io/unhealthy" ?